utils/etc/ansible.cfg

```diff
@@ -19,10 +19,15 @@ nocows = 1
 
 retry_files_enabled = False
 
+deprecation_warnings=False
```

Comment:
Is it not possible to filter these out in the callback plugin? Maybe it's not a big deal for the quick installer, but I know we are getting ready to hit a point where at least some of the deprecation warnings are going to be failures with a newer 2.2 build.